templates/dashboard/product/detail.html

```diff
@@ -231,6 +231,39 @@ <h6>
                 </table>
               </div>
             </div>
+      {% if no_variants %}
```

Comment:
1. Could you keep the same contexts as the variant's table to avoid duplicating strings?

https://github.com/mirumee/saleor/blob/a54c91605bd0def1eb052bdbef83c16103d2adab/templates/dashboard/product/product_variant/detail.html#L145-L173

2. Please keep the code aligned from the rest (as said before, just a reminder);
3. Instead of duplicating all this code, I think it would be best to create a new snippet and use a include instead (using the `with` keyword) in variant details template, and for this one here.